metadatastore/core.py

```diff
@@ -357,7 +357,8 @@ def get_events_generator(descriptor, event_col, descriptor_col,
                                       run_start_cache)
     col = event_col
     ev_cur = col.find({'descriptor': descriptor_uid},
-                      sort=[('time', pymongo.ASCENDING)])
+                      sort=[('descriptor', pymongo.DESCENDING),
```

Comment:
I do not think this should have the `'descriptor'` in the sort as now the time is essentially not used.